cmd/commands/runtime.go

```diff
@@ -1569,7 +1592,10 @@ func NewRuntimeUninstallCommand() *cobra.Command {
 	cmd.Flags().BoolVar(&opts.FastExit, "fast-exit", false, "If true, will not wait for deletion of cluster resources. This means that full resource deletion will not be verified")
 	cmd.Flags().BoolVar(&opts.DisableTelemetry, "disable-telemetry", false, "If true, will disable the analytics reporting for the uninstall process")
 
-	opts.CloneOpts = apu.AddCloneFlags(cmd, &apu.CloneFlagsOptions{CloneForWrite: true})
+	opts.CloneOpts = apu.AddCloneFlags(cmd, &apu.CloneFlagsOptions{
+		CloneForWrite: true,
+		Optional:      true,
```

Comment:
If we changed that to optional I expected to see somewhere in the code
```
if not managed:
  verify that clone opts were provided
```
otherwise you just let people with hybrid runtimes to get some weird error

Reply:
i added some checks for repo and git_token